core/dns.go

```diff
@@ -14,6 +14,35 @@ import (
 	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/miekg/dns"
 )
 
+type AddrFilter int
+
+const (
+	// NoAddrFilter is used to tell LookupHost to query both A and AAAA records
+	noAddrFilter AddrFilter = iota
```

Comment:
These should be publicized since they are in the public API in LookupHost. I know you want to use the NameToFilter for that, but it's hard to know that "" is what people should be using when they call it. This also avoids the core.AddrFilter(0) problem in the tests.